blues/simulation.py

```diff
@@ -77,6 +78,17 @@ def __init__(self, simulations, model, mover, **opt):
             self.verbose = opt['verbose']
         else:
             self.verbose = False
+        if 'write_ncmc' in opt:
+            self.write_ncmc = opt['write_ncmc']
+            if 'ncmc_outfile' in opt:
+                self.ncmc_outfile = opt['ncmc_outfile']
+            else: 
+                self.ncmc_outfile = 'ncmc_output.dcd'
+            
+            self.ncmc_reporter = app.dcdreporter.DCDReporter(self.ncmc_outfile, 1) 
```

Comment:
Could we move these into it's own class? That way we don't have reporters being added in two different places. This way we can easily expand it to support writing to other formats if we would like.